R/update-news.R

```diff
@@ -125,3 +148,26 @@ edit_cran_comments <- function() {
   local_options(usethis.quiet = TRUE)
   edit_file("cran-comments.md")
 }
+
+translate_type <- function(type) {
+  if (type %in% conventional_commit_types()) {
+    names(conventional_commit_types())[conventional_commit_types() == type]
+  } else {
+    type
+  }
```

Comment:
If we swap names and values in `conventional_commit_types()`:

```suggestion
   conventional_commit_types()[type] %|% type
```